### DISCUSSION TITLE
Frontend Redesign

### DISCUSSION THREAD

**Discussion by egekorkan** [thingweb/thingweb-playground#456] [Ideas]
Since it is a work in progress, it is not ready for a PR but here is an interactive version of the new UI that @SergioCasCeb is working on:

https://td-playground.netlify.app/

Only the super basic functionality is working but it is something for people to have a look at.

**Comment by SergioCasCeb** [comment]
The link has been changed to:  https://td-playground-beta.netlify.app/
